platforms/karaf/itests/src/test/java/io/fabric8/kubernetes/karaf/itests/FeatureInstallationTest.java

```diff
@@ -43,6 +43,7 @@ public class FeatureInstallationTest extends TestBase {
   @Test
   public void testKubernetesClientFeature() throws Exception {
     featuresService.addRepository(getFeaturesFile().toURI());
+    featuresService.installFeature("scr");
```

Comment:
After migrating to OSGi Component annotations from Felix SCR annotations, I see this additional text added in the MANIFEST.MF `Require-Capability`
```
filter:="(&(osgi.extender=osgi.compo
 nent)(version>=1.5.0)(!(version>=2.0.0)))
```

This causes karaf tests to fail with missing requirement. Once I install `scr` feature test passes.

Reply:
I moved it as a dependency feature in `feature.xml`